src/modules/sign/components/Sign.jsx

```diff
@@ -1,7 +1 @@
-import React from 'react';
-
-function Sign() {
-	return <div></div>;
-}
-
-export default Sign;
+import React from "react";

function Sign() {

	return <div></div>;
}

export default Sign;

```

Comment:
Why is the code mangled? Is this a formatting issue?

Reply:
@toystars probably a formatting issue, I will check that also